docs/lineage.rst

```diff
@@ -0,0 +1,87 @@
+Lineage
+=======
+
+Airflow can help track origins of data, what happens to it and where it moves over time. This can aid having
+audit trails and data governance, but also debugging of data flows.
+
+Airflow tracks data by means of inlets and outlets of the tasks. Let's work from an example and see how it
+works.
+
+.. code:: python
+
+    """
```

Comment:
Missing closing quotes of the comments? Maybe move the comments outside of the code block